src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), transaction.getQuantity());
```

Comment:
When implementing the Balance operation, ensure that you check the result balance before saving it in the Storage. It should always be positive. If the balance is negative, throw a `RuntimeException` to handle this case properly.